src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -14,5 +14,7 @@
     public static final String MESSAGE_INVALID_APPOINTMENT_DISPLAYED_INDEX = "The appointment index "
             + "provided is invalid";
     public static final String MESSAGE_MORE_THAN_ONE_SAME_PREFIX = "There should only be 1 option";
+    public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String MESSAGE_RECEIVED_MORE_THAN_INDEX = "Preamble contains more than just index.";
```

Comment:
Will users understand what preamble is referring to? Maybe we could rephrase it a bit

Reply:
This error message shouldn't be displayed to the users, its for the parser's to determine which error message to show.